homeassistant/components/guardian/strings.json

```diff
@@ -17,5 +17,11 @@
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     }
+  },
+  "issues": {
+    "deprecated_service": {
+      "title": "The {deprecated_service} service is deprecated",
+      "description": "This service is deprecated. Use the `{alternate_service}` service and pass it a target entity ID of `{alternate_target}` instead."
```

Comment:
We want to avoid the term: "Deprecated", to make it less technical in the UI. Instead use "is being removed", "is going to be removed" and similar.

Also, tell them what to do to resolve the issue.
E.g., update the automation or script that use this service and click the submit button below to mark this issue resolved (or similar).